.gitignore

```diff
@@ -1 +1,2 @@
-.precomp
+**/.precomp/
+
```

Comment:
This is the third time you make this change. I think your fork is not synced up with all the latest changes when you start working on your PRs.

`.precomp` alone already blocks `.precomp` name in *all* subdirectories. Your change just breaks that behaviour on systems where `**/` doesn't get expanded to mean all subdirs.

Reply:
That line is interpreted by git, not by the system. I did add it because .precomp did show up in subdirectories.

Reply:
Then just keep both of them in. I know the `**/` in `.gitignore` doesn't work on my ubuntu box.

Reply:
Those are 2.16.3 docs. Not all repos have latest and greatest. I don't see anything in those docs that'd explain why simple `.precomp` pattern doesn't work for you.

```
zoffix@~/R/rakudo (master)$ mkdir src/.precomp
zoffix@~/R/rakudo (master)$ gd
diff --git a/.gitignore b/.gitignore
index 3d1a43d..efb9f5d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -100,7 +100,7 @@ perl6-gdb-m
 perl6-valgrind-m
 *.rakudo.moar
 *.rakudo.jvm
-.precomp
+**/.precomp
 t/03-jvm/*.class
 /zef
 .DS_Store
zoffix@~/R/rakudo (master)$ gs
# On branch master
# Changes not staged for commit:
#   (use "git add <file>..." to update what will be committed)
#   (use "git checkout -- <file>..." to discard changes in working directory)
#
#       modified:   .gitignore
#
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#       .precomp/
#       t/02-rakudo/test-packages/.precomp/
#       t/04-nativecall/.precomp/
#       t/09-moar/.precomp/
#       t/packages/.precomp/
no changes added to commit (use "git add" and/or "git commit -a")
zoffix@~/R/rakudo (master)$ git --version
git version 1.7.9.5
zoffix@~/R/rakudo (master)$
```